lib/manager/gradle/index.js

```diff
@@ -0,0 +1,50 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const gradle = require('./build-gradle');
+
+function extractDependencies(content, fileName, config) {
+  logger.debug('gradle.extractDependencies()');
+  const deps = [{ name: 'dummy' }];
+  const gradleFile = `${config.localDir}/${fileName}`;
+
+  fs.writeFileSync(gradleFile, content);
+  execSync('gradle dependencyUpdates -Drevision=release', {
+    stdio: 'inherit',
+    cwd: config.localDir,
+  });
+
+  return { deps };
+}
+
+function getPackageUpdates(config) {
+  logger.debug('gradle.getPackageUpdates()');
+
+  const contents = fs.readFileSync(`${config.localDir}/build/dependencyUpdates/report.json`).toString();
+  const dependencies = JSON.parse(contents);
+
+  dependencies.outdated.dependencies.forEach(dep => {
```

Comment:
not sure why I have to do this. It seems that the group property is not propagated to the updateDependencies function, so I had to copy it with a different name